tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/inspections/VersionMismatchNotificationProvider.java

```diff
@@ -84,7 +84,7 @@ private static String getBallerinaPluginVersion() {
 
     @NotNull
     private String getMinorVersion(@NotNull String version) {
-        return version.contains(".") ? version.split("[.]")[1] : "";
+        return version.contains(".") ? version.replace('-','.').split("[.]")[1] : "";
```

Comment:
```suggestion
        return version.contains(".") ? version.replace('-', '.').split("[.]")[1] : "";
```

Don't we need a space? Is the gradle checkstyle plugin is running?

Reply:
It was disabled locally due a config change. Fixed the checkstyle issue